::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2017 Mirantis Inc., Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Sat, 5 Aug 2017 02:03:01 +0200
Subject: [PATCH] linux.network.interface: noifupdown support

According to [1], salt states/network supports "noifupdown" as a
parameter for each interface.
Adjust salt formula part for `linux.network.interface`, by extending
the "interface_params" in <linux/map.jinja> accordingly.

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Signed-off-by: Charalampos Kominos <Charalampos.Kominos@enea.com>
Signed-off-by: Guillermo Herrero <Guillermo.Herrero@enea.com>
---

diff --git a/linux/map.jinja b/linux/map.jinja
--- a/linux/map.jinja
+++ b/linux/map.jinja
@@ -92,6 +92,7 @@
     'hashing-algorithm',
     'hardware-dma-ring-rx',
     'hwaddr',
+    'noifupdown',
 ] %}

 {% set network = salt['grains.filter_by']({
